src/view/view_element.py

```diff
@@ -2,10 +2,10 @@ class ViewElement:
     def __init__(self, view, identifier, x, y, width, height):
         self.identifier = identifier
         self.view = view
-        self.x = x
-        self.y = y
-        self.w = width
-        self.h = height
+        self.x = int(x)
+        self.y = int(y)
+        self.width = int(width)
+        self.height = int(height)
         self.events = {}
 
     def draw(self):
```

Comment:
Every mouse movement now creates a print with 'event not handled'.
Please comment out the print statement in event_handler or do other changes
Before it wasn't a problem but some change led to create a print statement for every mouse movement event.